managing-os/linux-hvm-tips.md

```diff
@@ -94,3 +94,10 @@ Qubes agents
 
 Linux Qubes agents are written primarily for PV qubes, but it is possible to run them also in a HVM qube.
 However some work may be required to achieve this. Check [this thread](https://groups.google.com/group/qubes-devel/browse_thread/thread/081df4a43e49e7a5).
+
+Driver malfunction after suspend
+------------
+
+Occasionally, some drivers (such as Intel wireless NICs) may not function properly after a suspend.
+To work around the issue, add the driver (`iwlmvm` and `iwlwifi`, for example) to
+`/rw/config/suspend-module-blacklist` in your `sys-net` qube.
```

Comment:
IIRC, @marmarek was recently thinking about pre-adding these.

Reply:
Here https://github.com/QubesOS/qubes-issues/issues/3486? I think that's what he was commenting on in https://github.com/QubesOS/qubes-doc/pull/548#issuecomment-362187816.

Reply:
I think it's this: https://github.com/QubesOS/qubes-core-agent-linux/commit/cfbc953

Reply:
Maybe an issue would suffice instead; please feel free to close this if so!

Reply:
And it's already done for R3.2 (qubes-core-agent >= 3.2.23)